docs/csharp/language-reference/builtin-types/record.md

```diff
@@ -53,12 +53,12 @@ You can use positional parameters to declare properties of a record and to initi
 
 When you use the positional syntax for property definition, the compiler creates:
 
-* A public init-only auto-implemented property for each positional parameter provided in the record declaration.
-  - For `record` types and `readonly record struct` types: An [init-only](../keywords/init.md) property can only be set in the constructor or by using a property initializer.
-  - For `record struct` types: A read-write property that can be set in a constructor, property initializer, or assignment after construction.
+* A public auto-implemented property for each positional parameter provided in the record declaration.
+  - For `record` types and `readonly record struct` types: An [init-only](../keywords/init.md) property.
+  - For `record struct` types: A read-write property.
 * A primary constructor whose parameters match the positional parameters on the record declaration.
 * For record struct types, a parameterless constructor that sets each field to its default value.
-* A `Deconstruct` method with an `out` parameter for each positional parameter provided in the record declaration. This method is provided only if there are two or more positional parameters. The method deconstructs properties defined by using positional syntax; it ignores properties that are defined by using standard property syntax.
+* A `Deconstruct` method with an `out` parameter for each positional parameter provided in the record declaration. The method deconstructs properties defined by using positional syntax; it ignores properties that are defined by using standard property syntax.
```

Comment:
I'm not sure it's a good idea to remove the messaging around the fact that positional records will only generate a `Deconstruct` when there are two or more positional props. Is that called out anywhere else?

```suggestion
* A `Deconstruct` method with an `out` parameter for each positional parameter provided in the record declaration. The method deconstructs properties defined by using positional syntax; it ignores properties that are defined by using standard property syntax. This method is provided only if there are two or more positional parameters.
```

Reply:
I removed it because it's incorrect. The compiler *does* generate a deconstruct for a record with only one member.